src/components/HomeInfo.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import { Button } from '../atoms/Button';
+import { Avatar } from '../atoms/Avatar';
+
+class HomeInfo extends React.Component {
+    render() {
+        return (
+            <div className="container-homeInfo">
+                <h3>subtitle</h3>
+                <h1>title to tilorem</h1>
+                <div className="container-messager">
+                    <Avatar />
+                    <span className="message">{this.props.children}</span>
+                </div>
+                <Button children={'Button Lor'} />
```

Comment:
You don't have to give `children` as an attribute. It's a _special_ prop that which value is anything you give _inside_ the component:

```diff
- <Button children="Button Lor" />
+ <Button>Button Lor</Button>
```

These two are exactly the same. Try to use the second one.